packages/analytics/analytics-chart/src/components/chart-types/StackedBarChart.vue

```diff
@@ -212,7 +212,7 @@ const axesTooltip = ref<AxesTooltipState>({
   top: '0px',
   left: '0px',
   text: '',
-  offset: 0,
+  offset: -50, // Avoids App Shell obstruction
```

Comment:
This is for the app shell overlap fix:
https://kongstrong.slack.com/archives/C01RF6STU2F/p1708592869580189